### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by danieleongari**
Improving the form, day by day, while using it.
Today I used it for adding ca. 30 cofs from papers p1937-p1946.

### CODE REVIEW

**Comment by ltalirz** [comment]
thanks for the improvements!
let me know when we should sit together to go through them (will work on some other stuff today).

**Comment by danieleongari** [comment]
I'm working little by little on this, while adding more COF in some "free time"!
One thing to remember is that I included manage_crystal (https://github.com/danieleongari/manage_crystal) to make the final formatting consistent to the others: a minimal CIF file that I could not achieve with ASE nor pymatgen. It not yet on Pypi, so I should make a release, when I have some time and add the dependency.

**Comment by danieleongari** [comment]
Please merge: I added some nice feature and I made it very practical to use. 
Still the most of the burden come from problematic CIFs (see README), but there are so many different possible problems that I don't think it is worth automating the cleaning in this package.

File: cofdb_submit/main.py [hunk 1]
```diff
@@ -234,32 +197,45 @@ def servable(self):
...
+        if intervals[0].dimtype == '2D':
             self.inp_dimensionality.value = '2D'
-            
-            # multiply cell if cell vectors are too short
+
+            # Check if it is correcly oriented, and extend to two layers if onyly one is present
+            z_min_thr = 6 #if less, it is likely a single layer
             cell_lengths = atoms.get_cell_lengths_and_angles()[0:3]
-            if any(cell_lengths < 4.5):
+            if min(cell_lengths[0],cell_lengths[1] < z_min_thr): # X or Y are perpendicular to a single layer
```
Comment (ltalirz) [inline cofdb_submit/main.py]: not sure this is what you want 
```
>>> 0,1 < 1
(0, False)
>>> min(0,1 < 1)
0
```
Also, it's not very readable. Let's go for this
```suggestion
            if cell_lengths[0] < z_min_thr or cell_lengths[1] < z_min_thr: # X or Y are perpendicular to a single layer
```

Comment (danieleongari) [reply cofdb_submit/main.py]: Right, I placed the bracket in the wrong position. 
I'm going to correct it and also comment the code a little more.

**Comment by ltalirz** [comment]
perhaps check whether some of the https://github.com/ltalirz/curated-cofs-submission/issues now can be closed

**Pull request merged**